docs/modules/ROOT/pages/explanation/fair-use-policy.adoc

```diff
@@ -0,0 +1,52 @@
+= Fair Use Policy
+
+== Memory
+
+We set a limit on the amount of memory you can consume per namespace to protect the {product} zone from resource abuse.
+See the xref:references/default-quota.adoc[Default Quota] documentation for details.
+
+We bill all memory usage.
+Additionally we bill memory reservations which exceed the usage.
+See https://products.docs.vshn.ch/products/appuio/cloud/pricing_2021-12-01.html[{product} pricing] for details.
+
+== CPU
+
+We set a limit on the amount of CPU you can consume per namespace to protect the {product} zone from resource abuse.
+See the xref:references/default-quota.adoc[Default Quota] documentation for details.
+
+We currently bill CPU usage or reservations which exceed the fair use ratio determined by the {product} zone's underlying compute resources.
+The fair use memory to CPU ratio for each zone is listed in the https://portal.appuio.cloud/zones[{product} portal].
+See https://products.docs.vshn.ch/products/appuio/cloud/pricing_2021-12-01.html[{product} pricing] for details.
```

Comment:
Side-note: is there a sane way to make a pricing meta link which transparently redirects to the currently live pricing and gets updated (semi-)automatically? Or do we have a link like that already? Obviously the url shortener would work, but updating that would get forgotten for sure.